app/gui/src/model/execution_context.rs

```diff
@@ -216,6 +217,78 @@ pub struct LocalCall {
 
 
 
+// ==============================
+// === QualifiedMethodPointer ===
+// ==============================
+
+/// A method pointer containing the qualified module and type names.
+#[derive(Clone, Debug, PartialEq)]
+pub struct QualifiedMethodPointer {
+    /// A module name containing the method.
+    pub module:          module::QualifiedName,
+    /// A type on which the method is defined.
+    pub defined_on_type: tp::QualifiedName,
+    /// A method name.
+    pub name:            String,
+}
+
+impl QualifiedMethodPointer {
+    /// Create a method pointer representing a module method.
+    pub fn module_method(module: module::QualifiedName, name: String) -> QualifiedMethodPointer {
+        QualifiedMethodPointer { module: module.clone(), defined_on_type: module.into(), name }
+    }
+    /// Tries to create a new method pointer from string components.
+    pub fn from_unqualified(
```

Comment:
Is this "from unqualified" or from "from qualified text"?

Reply:
Agree, that is the right name for the method